tikv,pd: add config of min-resolved-ts (#13836)
pingcap/docs: pd-configuration-file.md

>
> If you have upgraded your cluster from a TiDB 4.0 version to the current version, the behavior of `flow-round-by-digit` after the upgrading and the behavior of `trace-region-flow` before the upgrading are consistent by default. This means that if the value of `trace-region-flow` is false before the upgrading, the value of `flow-round-by-digit` after the upgrading is 127; if the value of `trace-region-flow` is `true` before the upgrading, the value of `flow-round-by-digit` after the upgrading is `3`.

### `min-resolved-ts-persistence-interval` <span class="version-mark">New in v6.0.0</span>

> + Determines the interval at which the minimum resolved timestamp is persistent to the PD. If this value is set to `0`, it means that the persistence is disabled.
> + Default value: `"0s"` before v6.3.0 and `"1s"` in v6.3.0 and later, which is the smallest positive value.
> + Minimum value: `0`
> + Unit: second
> 
> > **Note:**
> >
> > For clusters upgraded from v6.0.0~v6.2.0, the default value of `min-resolved-ts-persistence-interval` will not change after the upgrade, which means that it will remain `"0s"`. To enable this feature, you need to manually change the value of this configuration item.

Reviewer: ```suggestion
+ Determines the interval at which the minimum resolved timestamp is persistent to the PD. If this value is set to `0`, it means that the persistence is disabled.
+ Default value: Before v6.3.0, the default value is `"0s"`. Starting from v6.3.0, the default value is `"1s"`, which is the smallest positive value.
+ Minimum value: `0`
+ Unit: second

> **Note:**
>
> For clusters upgraded from v6.0.0~v6.2.0, the default value of `min-resolved-ts-persistence-interval` does not change, which means that it will remain `"0s"`. To enable this feature, you need to manually change the value of this configuration item.
```

> > For clusters upgraded from v6.0.0~v6.2.0, the default value of `min-resolved-ts-persistence-interval` does not change, which means that it will remain `"0s"`. To enable this feature, you need to manually change the value of this configuration item.

Reviewer 2: ```suggestion
> For clusters upgraded from v6.0.0~v6.2.0, the default value of `min-resolved-ts-persistence-interval` does not change after the upgrade, which means that it will remain `"0s"`. To enable this feature, you need to manually change the value of this configuration item.
```
